### ISSUE TITLE
If both pdf and epub are available, download only pdf

### ISSUE THREAD

**Issue by prkandel** [alexgand/springer_free_books#48]
I noticed that there are several books which have both pdf and epub available. The current script downloads both, creating a lot of duplicate files. It would be nice to only download pdf if epub and pdf both are available.

**Comment by sinnytk** [comment]
A simple mode variable along with a check whether to download PDF + EPUB or only PDF(if available.)

I can work on this if that's ok?

@alexgand

**Comment by prkandel** [comment]
Yeah, the change should be simple. Sounds good to me.

**Comment by alexgand** [comment]
@sinnytk yes, please, but make it simple.

**Comment by sinnytk** [comment]
@alexgand 
These are the different modes that I came up with. What do you reckon would be a better implementation, using command line arguments or asking for mode as input from the user?

```
0, download both (EPUB and PDF)
1, download PDF only (if not available, skip)
2, download EPUB only (if not available, skip)
3, prefer PDF (otherwise download EPUB)
4, prefer EPUB (otherwise download PDF)
```

**Comment by chaosAD** [comment]
Have a look at my PR #54, I have included command-line options to select PDF only, EPUB, only, or both.

**Comment by alexgand** [comment]
Implemented in PR #57, please try the new code, it is now much better!
